docker/Dockerfile

```diff
@@ -124,24 +124,6 @@ RUN apt-get update && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
-##################################################
-# Change the NCCL version to fix NVLink errors
-# This is required to train on Nvidia A100s in GCP
-##################################################
-RUN if [ -n "$CUDA_VERSION" ] ; then \
```

Comment:
Do we want to make this a conditional thing that can be controlled via a build arg?
I have a slight preference to leaving the code around, until GCP fixes the underlying issue.

Reply:
I think we're not sure if GCP still has the issue, someone's gotta take some time and try it out again... maybe it's been fixed.

I can keep the code here but commented out if that's preferrable?

Reply:
Let's remove it, this was a total hack and support isn't scalable longer term especially as NCCL versions update with subsequent CUDA versions.  We can consider adding it back if someone raises an issue but let's keep it simple and let this be use case driven.